api/Document.json

```diff
@@ -8786,10 +8790,10 @@
               "version_added": null
             },
             "safari": {
-              "version_added": null
+              "version_added": "10.1"
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/Pointer_Lock_API#browser_compatibility says 10.1 for the unprefixed API, but also that there was a webkit-prefixed one. Is that right, and did that fire a prefixed event that should be added here?

Reply:
I took a closer look into this one, and found that the data we have for `api.Element.requestPointerLock` is inaccurate.  There was no prefixed version for the function before Safari 10.1, or even in 10.1 itself.  Tested in SauceLabs in Safari 10.0, 10.1, and 9.0.

I'll write a PR to fix up its data.

Reply:
Would you mind folding these changes into https://github.com/mdn/browser-compat-data/pull/11164 for easier review?